wire-ios-data-model/Tests/OneOnOne/OneOnOneMigratorTests.swift

```diff
@@ -90,6 +90,7 @@ final class OneOnOneMigratorTests: XCTestCase {
 
         await syncContext.perform {
             XCTAssertEqual(mlsConversation.oneOnOneUser?.remoteIdentifier, userID.uuid)
+            XCTAssertEqual(mlsConversation.allMessages.count, 0, "no system message should be inserted")
```

Comment:
test that no mls system message are inserted when users didn't have a proteus conversation before